docs/Manpage.md

```diff
@@ -1,4 +1,4 @@
-brew(1) -- The missing package manager for macOS
+brew(1) -- The Missing Package Manager for macOS
```

Comment:
Would be good to update Homebrew/brew.sh accordingly too.

Reply:
https://github.com/Homebrew/brew.sh/pull/436